app/src/full/AndroidManifest.xml

```diff
@@ -5,11 +5,8 @@
 
     <application
         android:name=".HomeAssistantApplication" >
-        <meta-data android:name="io.sentry.auto-init" android:value="${sentryEnabled}" />
+        <meta-data android:name="io.sentry.auto-init" android:value="false" />
         <meta-data android:name="io.sentry.release" android:value="${sentryRelease}" />
-        <meta-data android:name="io.sentry.session-tracking.enable" android:value="true" />
-        <meta-data android:name="io.sentry.ndk.enable" android:value="false" />
```

Comment:
So this was added due to a specific crash in the Pixel 4 XL, I may need to test it locally and see if the crash continues or not with debug turned off.